frontend/src/metabase/selectors/settings.ts

```diff
@@ -107,3 +107,6 @@ export const getIsPaidPlan = createSelector(
     return tokenStatus != null && tokenStatus.valid;
   },
 );
+
+export const getLastAcknowledgedVersion = (state: State) =>
```

Comment:
We don't add simple selectors for settings here. If needed, add one in `nav` or in the component file